netbox_onboarding/netdev_keeper.py

```diff
@@ -257,27 +259,38 @@ def get_onboarding_facts(self):
 
             napalm_device.open()
 
-            logging.info("COLLECT: device facts")
+            logger.info("COLLECT: device facts")
             self.facts = napalm_device.get_facts()
 
-            logging.info("COLLECT: device interface IPs")
+            logger.info("COLLECT: device interface IPs")
             self.ip_ifs = napalm_device.get_interfaces_ip()
 
-            try:
-                module_name = PLUGIN_SETTINGS["onboarding_extensions_map"].get(self.napalm_driver)
+            module_name = PLUGIN_SETTINGS["onboarding_extensions_map"].get(self.napalm_driver)
+
+            if module_name:
                 module = importlib.import_module(module_name)
                 driver_addon_class = module.OnboardingDriverExtensions(napalm_device=napalm_device)
                 self.onboarding_class = driver_addon_class.onboarding_class
                 self.driver_addon_result = driver_addon_class.ext_result
-            except ImportError as exc:
-                logging.info("No onboarding extension found for driver %s", self.napalm_driver)
+
+            else:
+                logger.info(
+                    "INFO: No onboarding extension defined for napalm driver %s, using default napalm driver",
+                    self.napalm_driver,
+                )
 
         except ConnectionException as exc:
             raise OnboardException(reason="fail-login", message=exc.args[0])
 
         except CommandErrorException as exc:
             raise OnboardException(reason="fail-execute", message=exc.args[0])
 
+        except ModuleNotFoundError as exc:
```

Comment:
This one could still be applied to the import part only.
Why change was made from previous `ImportError` ?